openpype/tools/workfiles/window.py

```diff
@@ -94,6 +95,19 @@ def _on_save_click(self):
         self._on_note_change()
         self.save_clicked.emit()
 
+    def get_user_name(self, file):
+        """Get user name from file path"""
+        # Only run on Unix because pwd module is not available on Windows.
+        # NOTE: we tried adding "win32security" module but it was not working
+        # on all hosts so we decided to just support Linux until migration
+        # to Ayon
+        if platform.system().lower() == "window":
```

Comment:
And I've made a typo 😞 
```suggestion
        if platform.system().lower() == "windows":
```

Reply:
hehe, good catch!